### ISSUE TITLE
Cycling highway tags with way selected (`C`) causes stack trace

### ISSUE THREAD

**Issue by Bonkles** [facebookincubator/RapiD#527]
Cycle Highway tags (either via right-click menu or via the hotkey) does in fact work, but seems to generate stack traces when used from the context menu. 

![Image](https://user-images.githubusercontent.com/1887955/182906175-1deae3ec-b1e0-482d-8aa7-b1838e4de7d0.png)

**Comment by Bonkles** [comment]
Okay, the above stack trace is a lie that chrome 103 was telling us (v103 source maps are borked) and the issue was instead to do with getting a mode's operations - new modes have an operations array, old modes have an operations method. 


Here's the *real* stack trace: 

```
Uncaught TypeError TypeError: context2.mode(...).operations is not a function
    at ui.showEditMenu (/Users/clarkben/working/RapiD/modules/ui/init.js:605:79)
    at _showContextMenu (/Users/clarkben/working/RapiD/modules/behaviors/BehaviorSelect.js:501:25)
    at _pointerup (/Users/clarkben/working/RapiD/modules/behaviors/BehaviorSelect.js:248:14)
    at emit (/Users/clarkben/working/RapiD/node_modules/eventemitter3/index.js:202:33)
    at InteractionManager2.onPointerComplete (/Users/clarkben/working/RapiD/node_modules/@pixi/interaction/src/InteractionManager.ts:1242:18)
    at InteractionManager2.onPointerUp (/Users/clarkben/working/RapiD/node_modules/@pixi/interaction/src/InteractionManager.ts:1302:14)
```

A quick line of code to detect which was which was all this bug needed.

**Comment by Bonkles** [comment]
Fixed in commit 93a0e1c6a.
